openapi/data_repository_service.swagger.yaml

```diff
@@ -194,57 +194,71 @@ definitions:
           sha512
   Bundle:
     type: object
-    required: ['id', 'object_ids', 'created', 'updated', 'version', 'checksums']
     properties:
       id:
         type: string
-        description: |-
+        description: >-
           An identifier, unique to this Data Bundle
       object_ids:
         type: array
         items:
           type: string
-        description: |-
+        description: >-
           The list of Data Objects that this Data Bundle contains.
       created:
         type: string
         format: date-time
-        description: |-
+        description: >-
           Timestamp of object creation in RFC3339.
       updated:
         type: string
         format: date-time
-        description: |-
-          Timestamp of update in RFC3339, identical to create timestamp in systems
-          that do not support updates.
+        description: >-
+          Timestamp of update in RFC3339, identical to create timestamp in
+          systems that do not support updates.
       version:
         type: string
-        description: |-
-          A string representing a version, some systems may use checksum, a RFC3339
-          timestamp, or incrementing version number. For systems that do not support
-          versioning please use your update timestamp as your version.
+        description: >-
+          A string representing a version, some systems may use checksum, a
+          RFC3339 timestamp, or incrementing version number. For systems that do not
+          support versioning please use your update timestamp as your version.
+      mime-type:
+        type: string
+        description: >-
+          A string providing the mime-type of the Data Bundle.
+          For example, "application/json".
       checksums:
         type: array
-        items:
-          $ref: '#/definitions/Checksum'
-        description: |-
+        description: >-
           At least one checksum must be provided.
           The Data Bundle checksum is computed over all the checksums of the
           Data Objects that bundle contains.
+        items:
+          $ref: '#/definitions/Checksum'
+      access_methods:
+        type: array
+        items:
+          $ref: '#/definitions/AccessMethod'
       description:
         type: string
-        description: |-
-          A human readable description.
+        description: A human readable description.
       aliases:
         type: array
+        description: A list of strings that can be used to identify this Data Bundle.
         items:
           type: string
-        description: |-
-          A list of strings that can be used to identify this Data Bundle.
-      system_metadata:
-        $ref: '#/definitions/SystemMetadata'
-      user_metadata:
-        $ref: '#/definitions/UserMetadata'
+      contents:
+        type: array
+        items:
+          $ref: '#/definitions/BundleObjects'
+    required:
+      - id
+      - size
```

Comment:
this object doesn't have a `size` field

Reply:
I missed this from the object. It should have a `size` computed from the cumulative sizes of each Data Object

Reply:
Hmm -- that means all implementors would need to compute the total size for all objects in a bundle before they could expose it via DRS. I guess that's okay, given that we also require them to compute the checksum, but I'd like to hear from at least a couple people planning to implement Bundle support before imposing that mandate.